include/quda.h

```diff
@@ -334,6 +334,38 @@ extern "C" {
     /** Whether to use additive or multiplicative Schwarz preconditioning */
     QudaSchwarzType schwarz_type;
 
+    /**
```

Comment:
These parameters need to be mirrored to quda_fortran.F90

Reply:
https://github.com/lattice/quda/pull/1061/commits/2769f6268cd7367667fc6c2b38cde0a44442f28c. Can you take a look and make sure it works with fortran?

Reply:
I've verified that the Fortran interface is working fine with this change.  Thanks.